lib/Model/IMAPMessage.php

```diff
@@ -724,14 +724,17 @@ public function toDbMessage(int $mailboxId, MailAccount $account): Message {
 		$msg->setFlagMdnsent(in_array(Horde_Imap_Client::FLAG_MDNSENT, $flags, true));
 
 		$allowed = [
-			Horde_Imap_Client::FLAG_SEEN,
 			Horde_Imap_Client::FLAG_ANSWERED,
 			Horde_Imap_Client::FLAG_FLAGGED,
+			Horde_Imap_Client::FLAG_FORWARDED,
 			Horde_Imap_Client::FLAG_DELETED,
 			Horde_Imap_Client::FLAG_DRAFT,
-			Horde_Imap_Client::FLAG_RECENT,
 			Horde_Imap_Client::FLAG_JUNK,
+			Horde_Imap_Client::FLAG_NOTJUNK,
 			Horde_Imap_Client::FLAG_MDNSENT,
+			Horde_Imap_Client::FLAG_RECENT,
```

Comment:
nope nope nope :)

Reply:
That shouldn't even be there! :grimacing: